src/param/param_type.rs

```diff
@@ -4,65 +4,61 @@ use std::fmt;
 #[derive(Debug, Copy, Clone, PartialEq)]
 #[repr(C)]
 pub(crate) enum ParamType {
-    HeartbeatInfo = 1,
```

Comment:
could you tell why you removed the numbers here? this will be a breaking change, I think

Reply:
This shouldn't be breaking because the enum is only pub(crate)

Reply:
> This shouldn't be breaking because the enum is only pub(crate).

Yeah, but the fields are being embedded into Headers. So

BEFORE: header.typ was 8 (e.g. in case of UnrecognizedParam)
AFTER: header.typ is 5 (e.g. in case of UnrecognizedParam)

and `fn from(v: u16) -> ParamType` implementation is using the old numbers => I don't think it will work correctly either

Reply:
I think everything is okay. Thanks